tests/clickhouse/test_formatter.py

```diff
@@ -15,7 +17,6 @@
     (Literal(None, 123), "123",),  # INT literal
     (Literal(None, 123.321), "123.321",),  # FLOAT literal
     (Literal(None, None), "NULL",),  # NULL
-    (Literal(None, True), "true",),  # True
```

Comment:
Was this an intentional removal? This seems unrelated.

Reply:
No it is not. Actually I am not sure how I did that.